nlp-model.py

```diff
@@ -0,0 +1,52 @@
+import re
+from nltk.stem import WordNetLemmatizer
+nltk.download('wordnet')
+lemmatizer = WordNetLemmatizer()
+
+
+class PreprocessText(object):
+    """
+    Sentiment relevant text properties. 
+    """
+    
+    def __init__(self, text):
+        
+        self.text = text
+        self.clean_text = self._clean_text()
+        
+    def _clean_text(self):
+        
+        wordz = self.text.split()
+        
+        lemmatizer = WordNetLemmatizer()
+        words = []
+        for word in wordz:
+            words.append(lemmatizer.lemmatize(word))
+        
+        return words
+
+class Sentiment(object):
+    """
+    Sentiment Analyzer
+    """
+    
+    
+    def __init__(self, lexicon):
+        
+        self.lexicon = lexicon
+        
+    def score(self, text):
+        
+        words = PreprocessText(text).clean_text
```

Comment:
Either this should be renamed to call `_clean_text` or the method `_clean_text` above should be renamed to `clean_text`. Probably the latter since a leading `_` in a method name denotes that it's not used anywhere else, and it seems to be used here

```suggestion
        words = PreprocessText(text)._clean_text
```